Makefile

```diff
@@ -177,10 +177,21 @@ test-odo-login-e2e:
 test-odo-config:
 	ginkgo -v -nodes=$(TEST_EXEC_NODES) -focus="odo config test" slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -randomizeAllSpecs  tests/integration/ -timeout $(TIMEOUT)
 
-# Run all integration tests
-.PHONY: test-integration
-test-integration:
-	go test -v github.com/openshift/odo/tests/integration -ginkgo.slowSpecThreshold=$(SLOW_SPEC_THRESHOLD) -ginkgo.v -timeout $(TIMEOUT)
+# Runs all integration test irrespective of service catalog status in the cluster.
+# Service and link command tests are not the part of this test run
+.PHONY: test-integration-all
+test-integration-all:
```

Comment:
The make target has `all` in his name but it is not running all the test. This will be confusing.

Reply:
Right, that is why i added in the comment section what test are be the part of this run. Even i covered a bit description in doc too - https://github.com/openshift/odo/pull/1912/files#diff-c26dca7a0015d5429db3d986bc160252R387

I will remove all from the name

Reply:
Done